modules/settings/assets/js/api/index.js

```diff
@@ -0,0 +1,132 @@
+import apiFetch from '@wordpress/api-fetch';
+import { addQueryArgs } from '@wordpress/url';
+import APIError from './exceptions/APIError';
+
+const wpV2Prefix = '/wp/v2';
+const v1Prefix = '/site-mailer/v1';
```

Comment:
I assume it also shouldn't be `/site-mailer/`? 
Ignore it though if you fixed it in some other PR.

Reply:
@pkniazevych updated in other PRs